lib/tasks/ood-proxy.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require_relative 'build_utils'
+include BuildUtils
+
+namespace :build do
+  def proxy_container_name
+    'ood-proxy-rs' || ENV['OOD_PROXY_CONTAINER_NAME'].to_s
+  end
+
+  task :proxy_docker do
+    sh build_cmd("Dockerfile.proxy", proxy_container_name)
+    sh tag_latest_container_cmd(proxy_container_name)
+  end
+
+  task :proxy do
+    build_args = ['cargo', 'build']
```

Comment:
Just wanted to point out that having a direct build Rake task will work very well with the RPM build.  For EL7 builds the `cargo` package comes from EPEL and for EL8 builds it comes from the OS.